canary/producer-c/CMake/Dependencies/libkvsProducerC-CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ include(ExternalProject)
 
 ExternalProject_Add(libkvsProducerC-download
     GIT_REPOSITORY    https://github.com/awslabs/amazon-kinesis-video-streams-producer-c.git
-    GIT_TAG           4bd762510e3b84cc310058fd238e0748681e2e80
+    GIT_TAG           7feaaba0cab44b415ee279fca515dd967c58c453
```

Comment:
I just merged your producer-c PR, the commit hash on develop is:

`18ab7ffc0e8f078f586d4e249dc6907b0366b8eb`

Reply:
Yes. Just modified it.